src/pages/home/HeaderView.js

```diff
@@ -92,7 +92,7 @@ function HeaderView(props) {
     const {translate} = useLocalize();
     const theme = useTheme();
     const styles = useThemeStyles();
-    const participants = lodashGet(props.report, 'participantAccountIDs', []);
+    const participants = ReportUtils.isGroupChat(props.report) ? ReportUtils.getVisibleMemberIDs(props.report) : lodashGet(props.report, 'participantAccountIDs', []);
```

Comment:
Can we simply call getParticipantsIDs() here?

Reply:
for money request report getParticipantsIDs adds managerID and ownerAccountID to participantAccountIDs that's the difference and the reason I used getVisibleMemberIDs is b/c getParticipantsIDs is getting deprecated

Reply:
Okay, I mean why not use 
`const participants = ReportUtils.getVisibleMemberIDs(props.report);` simply?
It works for all cases, no?

Reply:
Yep Done!!